CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.6.3)
+project(kvssdp C)
```

Comment:
Do we need to set C standard?

Reply:
I just removed this language to use default setting (C/CXX).
I can still pass test cases with that.